### PR TITLE
feat(aws-eventbridge-kinesisfirehose-s3) created new construct

### PR DESCRIPTION
**Pull request by mickychetta**
*Issue #288, if available:*

*Description of changes:*
-Created new construct for aws-eventbridge-kinesisfirehose-s3
-Created wrapper in aws-events-rule-kinesisfirehose-s3 to support new construct

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-fkVQbXRiQi6A
* Commit ID: 49942bbb118e77dfe62875a6115431dad1523300
* Result: SUCCEEDED
* [Build Logs](https://nis3u8ftb6.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=a3e58617-7982-4119-8b09-aaa7aabbf302%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: source/patterns/@aws-solutions-constructs/aws-eventbridge-kinesisfirehose-s3/lib/index.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+"use strict";
```
Comment (biffgaut) [inline source/patterns/@aws-solutions-constructs/aws-eventbridge-kinesisfirehose-s3/lib/index.js:1]: This is a build product, it shouldn't have gotten checked in. Check your .gitignore

File: source/patterns/@aws-solutions-constructs/aws-eventbridge-kinesisfirehose-s3/test/eventbridge-kinesisfirehose-s3.test.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
+"use strict";
```
Comment (biffgaut) [inline source/patterns/@aws-solutions-constructs/aws-eventbridge-kinesisfirehose-s3/test/eventbridge-kinesisfirehose-s3.test.js:1]: Build product, shouldn't be in git. Seems like you're miss *.js in your .gitignore file. The original .gitignore starts with:

```
lib/*.js
test/*.js
!test/lambda/*
*.js.map
```

This construct is missing the first 3 lines. The 3rd line is not needed as there is no test/lambda folder in this construct (I think), but the first 2 lines are very neccessary.

File: source/patterns/@aws-solutions-constructs/aws-eventbridge-kinesisfirehose-s3/test/integ.eventbridge-kinesisfirehose-s3-no-arguments.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+"use strict";
```
Comment (biffgaut) [inline source/patterns/@aws-solutions-constructs/aws-eventbridge-kinesisfirehose-s3/test/integ.eventbridge-kinesisfirehose-s3-no-arguments.js:1]: Nope, shouldn't be here

File: source/patterns/@aws-solutions-constructs/aws-events-rule-kinesisfirehose-s3/.gitignore [hunk 4]
```diff
@@ -1,6 +1,5 @@
 lib/*.js
 test/*.js
-!test/lambda/*
```
Comment (biffgaut) [inline source/patterns/@aws-solutions-constructs/aws-events-rule-kinesisfirehose-s3/.gitignore:3]: Yes, like this. Just remove this line.

**Review by biffgaut — CHANGES_REQUESTED** [review]
I think the comments all come down to 2 missing lines in .gitignore

**Review by biffgaut — APPROVED** [review]

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-fkVQbXRiQi6A
* Commit ID: d90e8a5fe9721a8258909a16aaa820d2ccaf734c
* Result: SUCCEEDED
* [Build Logs](https://nis3u8ftb6.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=e147f815-92b3-44a9-8aba-dea9f436ca48%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-fkVQbXRiQi6A
* Commit ID: 2680e425b001ed7a279b4aee265f35f6455ec194
* Result: SUCCEEDED
* [Build Logs](https://nis3u8ftb6.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=2b596e56-77ba-4846-9ed3-a4fc0e3500bb%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: codebuildgithubautobuildPro-fkVQbXRiQi6A
* Commit ID: eb5a996f818a262269e0b348a17e648153692974
* Result: SUCCEEDED
* [Build Logs](https://nis3u8ftb6.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=608b3112-568d-4971-b369-4a5ad5be82a4%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**